Importing @remix-project/remix-simulator in vite project fails

```
✘ [ERROR] Could not resolve "tslib"

    node_modules/.pnpm/@remix-project+remix-simulator@0.2.36_typescript@5.3.2/node_modules/@remix-project/remix-simulator/src/provider.js:4:24:
      4 │ const tslib_1 = require("tslib");
        ╵                         ~~~~~~~

  You can mark the path "tslib" as external to exclude it from the bundle, which will remove this
  error and leave the unresolved path in the bundle. You can also surround this "require" call with
  a try/catch block to handle this failure at run-time instead of bundle-time.

```

Reproduction repo: https://github.com/wighawag/ethereum-remix-provider

```
pnpm i
pnpm dev
```

--- a/libs/remix-simulator/package.json
+++ b/libs/remix-simulator/package.json
@@ -37,6 +37,7 @@
     "rlp": "^3.0.0",
     "string-similarity": "^4.0.4",
     "time-stamp": "^2.0.0",
+    "tslib": "^2.3.0",
     "web3": "^4.1.1",
     "web3-utils": "^4.0.5"
   },

